packages/fxa-content-server/app/scripts/lib/metrics.js

```diff
@@ -383,8 +401,10 @@ _.extend(Metrics.prototype, Backbone.Events, {
       marketing: flattenHashIntoArrayOfObjects(this._marketingImpressions),
       migration: this._migration,
       numStoredAccounts: this._numStoredAccounts,
-      plan_id: this._planId,
-      product_id: this._productId,
+      // planId and productId are optional so we can physically remove
+      // them from the payload instead of sending NOT_REPORTED_VALUE
+      planId: this._subscriptionModel.get('planId') || undefined,
+      productId: this._subscriptionModel.get('productId') || undefined,
```

Comment:
`|| undefined` is to overwrite `null`, because `undefined` won't be serialised so saves a little bit of space in the payload.